src/core/tests/frontend/paddle/test_models/gen_scripts/generate_sum.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import print_function
+import sys
+import numpy as np
+import paddle
+
+from save_model import saveModel
+
+
+def sum_(name: str, input):
+    paddle.enable_static()
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        data = paddle.static.data('data', shape=input.shape, dtype=input.dtype)
+        out = paddle.fluid.layers.sum(data)
```

Comment:
```suggestion
        out = paddle.add_n(data)
```
recommend we also use 'paddle.add_n()' here to align with the used way of the following 'sum(name: str, inputs)'.
other parts LGTM

Reply:
Done